app/controllers/register_controller.rb

```diff
@@ -14,16 +14,16 @@ def index
     @changes = Change.joins('LEFT OUTER JOIN statuses on statuses.change_id = changes.id')
                      .where("register_name = '#{params[:register_id]}' AND statuses.status = 'pending'")
 
-    @register = get_register(params[:register_id])
-                ._all_records
-                .sort_by(&:key)
+    @register = @registers_client.get_register(params[:register_id], Rails.configuration.register_phase, nil)
```

Comment:
Can we not specify the`nil` parameter? I believe this is the default value for the `data_store` parameter in the `get_register` method anyway?